manifest.json

```diff
@@ -11,7 +11,8 @@
   ],
   "settingsSchema": {},
   "dependencies": {
-    "vtex.styleguide": "6.x"
+    "vtex.styleguide": "6.x",
```

Comment:
7.x

Reply:
Fixed in commit 2d2092c522f572a871eea9c57d10a73b4837ee5f